serve.go

```diff
@@ -18,22 +16,30 @@ import (
 
 func Serve(c *util.Config) {
 	endpoint := c.Endpoint
+	health, err := newHealthHandler(c)
+	if err != nil {
+		log.Fatalf("Error: %v", err)
```

Comment:
This exits 1 after writing the output.

Reply:
Since this is being refactored, I would say any os.Exit() or log.Fatalf should be in `cmd/goss/goss.go`

@ripienaar did a refactor a while back and moved all exits up to the top layer where it belonged.

Reply:
Done in fb6fa35.